pyproject.toml

```diff
@@ -1,70 +1,69 @@
 [project]
-requires-python = ">=3.8"
-version = "2.0.1"
+requires-python = ">=3.8,<4.0"
 name = "astrapy"
-description = "AstraPy is a Pythonic SDK for DataStax Astra and its Data API"
+version = "2.0.1"
+description = "A Python client for the Data API on DataStax Astra DB"
 authors = [
     {"name" = "Stefano Lottini", "email" = "stefano.lottini@datastax.com"},
     {"name" = "Eric Hare", "email" = "eric.hare@datastax.com"},
 ]
-license = "Apache-2.0"
 readme = "README.md"
-keywords = ["DataStax", "Astra"]
-homepage = "https://github.com/datastax/astrapy"
-repository = "https://github.com/datastax/astrapy"
+
+keywords = ["DataStax", "Astra DB", "Astra"]
+dependencies = [
+    "deprecation ~= 2.1.0",
+    "httpx[http2]>=0.25.2,<1",
+    "h11 >= 0.16.0",
+    "pymongo >= 3",
+    "toml >= 0.10.2,<1.0.0",
+    "typing-extensions >= 4.0",
+    "uuid6 >= 2024.1.12"
+]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-    "Topic :: Software Development :: Build Tools",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Topic :: Software Development :: Build Tools"
 ]
 
-[tool.poetry]
-version = "2.0.1"
-name = "astrapy"
-description = "AstraPy is a Pythonic SDK for DataStax Astra and its Data API"
-authors = [
-    "Stefano Lottini <stefano.lottini@datastax.com>",
-    "Eric Hare eric.hare@datastax.com>",
-]
-readme = "README.md"
-packages = [
-    { include = "astrapy" }
-]
+[project.urls]
+Homepage = "https://github.com/datastax/astrapy"
+Documentation = "https://docs.datastax.com/en/astra-db-serverless/api-reference/dataapiclient.html"
+Repository = "https://github.com/datastax/astrapy"
+Issues = "https://github.com/datastax/astrapy/issues"
+Changelog = "https://github.com/datastax/astrapy/blob/main/CHANGES"
 
-[tool.poetry.dependencies]
-python = "^3.8.0"
-deprecation = "~2.1.0"
-h11 = ">=0.16.0"
-httpx = { version=">=0.25.2,<1", extras=["http2"] }
-pymongo = ">=3"
-toml = "^0.10.2"
-typing-extensions = ">=4.0"
-uuid6 = ">=2024.1.12"
-
-[tool.poetry.group.dev.dependencies]
-faker = "~23.1.0"
-mypy = "~1.9.0"
-pre-commit = "~3.5.0"
-pytest-asyncio = "~0.23.5"
-pytest-cov = "~4.1.0"
-pytest-testdox = "~3.1.0"
-pytest = "~8.0.0"
-python-dotenv = "~1.0.1"
-pytest-httpserver = "~1.0.8"
-testcontainers = "~3.7.1"
-ruff = "^0.6.8"
-types-toml = "^0.10.8.7"
-blockbuster = "~1.5.5"
+[project.optional-dependencies]
+dev = [
```

Comment:
It would be better to put dev dependencies in the dev group.
See https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies
Then when you install, you do it with `uv sync --dev`